tauri/src/settings/mod.rs

```diff
@@ -80,6 +80,7 @@ pub struct SerializedSettings {
 
     pub abi_watch: bool,
     pub abi_watch_path: Option<String>,
+    pub hide_empty_tokens: bool,
```

Comment:
```suggestion
    #[serde(default = true)]
    pub hide_empty_tokens: bool,
```

this needs to be done so that existing installations still work when deserializing an existing settings file